src/v/security/audit/audit_log_manager.h

```diff
@@ -126,6 +128,20 @@ class audit_log_manager
           make_api_activity_event(std::forward<Args>(args)..., {})));
     }
 
+    template<typename... Args>
```

Comment:
I think we have this issue in another method in this file, but if you use `std::forward` on `args` then `Args...` must be a universal ref (`Args&&...`)